rosidl_generator_c/rosidl_generator_c/__init__.py

```diff
@@ -87,6 +87,10 @@ def idl_structure_type_to_c_include_prefix(namespaced_type, subdirectory=None):
         include_prefix = include_prefix[:-8]
     elif include_prefix.endswith('__feedback'):
         include_prefix = include_prefix[:-10]
+    elif include_prefix.endswith('__send_goal'):
+        include_prefix = include_prefix[:-11]
+    elif include_prefix.endswith('__get_result'):
+        include_prefix = include_prefix[:-12]
```

Comment:
unrelated improvement, but I would prefer not counting characters:

```python3
suffixes_to_strip = [
  '__request',
  ....
  '__get_result'
]
for suf in suffixes_to_strip:
  if include_prefix.endswith(suf):
    return include_prefix[:-len(suf)]
```

Reply:
I suggest we do that kind of refactor in a separate PR.